README.md

```diff
@@ -23,9 +23,9 @@ Please also make sure your node version is higher than 8.x.
 If you have higher version of node installed, you may consider `nvm` to allow different versions of `node` coexisting on your machine.
 
 1. Follow the [instructions](http://nvm.sh) to install nvm
-2. Run `nvm install v8.16.0` to install node v8
-3. Run `nvm use v8.16.0` to switch the working environment to node v8
-4. Run ` npm i docsite@wuhan2020 -g ` to install docsite for project wuhan2020.
+2. Run `nvm install 10` to install node v8
+3. Run `nvm use 10` to switch the working environment to node v8
```

Comment:
It should be `v10` for the end of both lines

![image](https://user-images.githubusercontent.com/4352234/74596050-b0dd8980-5017-11ea-87d4-b313972659d2.png)

Reply:
Nice catch